CMakeLists.txt

```diff
@@ -11,7 +11,7 @@ add_subdirectory (CMake)
 
 ################################
 ## Version information
-set(HOOMD_VERSION_RAW "2.6.0")
+set(HOOMD_VERSION_RAW "3.0.0")
```

Comment:
This should read 2.8.1 currently. The `bumpversion` tool scans for the current version and automatically replaces it with the new version to be released.

Reply:
done